### PR TITLE
Ddpb 689 gds details

### PR DESCRIPTION
**Pull request by Biggsen**
custom accordian replaced with native <details>

### CODE REVIEW

File: Gulpfile.js [hunk 1]
```diff
@@ -187,7 +189,7 @@ gulp.task('lint.sass', function() {
 
 // Check javascript follows some good guidelines and check for obvious errors.
 gulp.task('lint.js', function () {
-    gulp.src(config.jsSrc + '/**/*.js')
+    gulp.src([config.jsSrc + '/**/*.js','!'+ config.jsSrc + '/**/details.polyfill.js'])
```
Comment (elvisciotti) [inline Gulpfile.js]: what's the semantic of "!" ?

File: src/AppBundle/Resources/assets/javascripts/modules/accounttransactions.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
+/* globals jQuery */
```
Comment (elvisciotti) [inline src/AppBundle/Resources/assets/javascripts/modules/accounttransactions.js]: I think this module can be deleted

File: Gulpfile.js (hunk 1 above)
Comment (Biggsen) [inline Gulpfile.js]: exclude

File: src/AppBundle/Resources/assets/javascripts/modules/accounttransactions.js (hunk 2 above)
Comment (Biggsen) [inline src/AppBundle/Resources/assets/javascripts/modules/accounttransactions.js]: all of JS needs looking at. probably loads we can throw out.

**Pull request merged**